### PR TITLE
Prefix renaming

### PR DESCRIPTION
**Pull request by kaelig**

### CODE REVIEW

**Comment by AlbertoElias** [comment]
Travis failing

**Comment by kaelig** [comment]
Thanks, looked like a Travis cache issue. It's now fixed!

File: demos/header-no-secondary.html [hunk 1]
```diff
@@ -3,16 +3,17 @@
...
+	<script src="//polyfill.webservices.ft.com/v1/polyfill.min.js?features=modernizr:classList,modernizr:customevent,modernizr:unknownelements,modernizr:es5array,modernizr:svg,modernizr:datauri,modernizr:cssgradients,modernizr:mediaqueries,modernizr:csstransitions,default"></script>
+	<style>body { margin: 0; } .demo-js .o--if-nojs { display: none; }</style>
+	<script>(function(d) { d.className = d.className + ' demo-js'; })(document.documentElement);</script>
+	<link rel="stylesheet" href="/bundles/css?modules=o-header:/demos/src/scss/demo.scss" />
 </head>
 <body class="o-hoverable-on">
-    <script>(function(d) {var b = d.getElementsByTagName('body')[0];b.className = b.className + ' demo-js';})(document);</script>
-    <header data-o-component="o-ft-header" class="o-ft-header o-registry-demo-chunk">
+<header data-o-component="o-ft-header" class="o-ft-header o-registry-demo-chunk">
 	<div class="o-ft-header__container">
```
Comment (AlbertoElias) [inline demos/header-no-secondary.html]: Classes are still o-ft-header

File: readme.md [hunk 2]
```diff
@@ -120,10 +122,10 @@ $o-ft-header-primary-featured-width
 
 ## Javascript instantiation
 
-An __o-ft-header__ object must be constructed for every `<header>` you have on your page that uses this module.
+An __o-header__ object must be constructed for every `<header>` you have on your page that uses this module.
 
 ```javascript
-var oFtHeader = require('o-ft-header');
+var oFtHeader = require('o-header');
```
Comment (AlbertoElias) [inline readme.md]: var oHeader?

Comment (kaelig) [inline readme.md]: Oh, right. makes sense I guess!

**Pull request merged**